WordPress/gutenberg
# Remove fill="none" from pinSmall icon

## What?

Remove fill="none" from pinSmall icon

## Why?

pinSmall Icon can't see in the Storybook.
see: https://wordpress.github.io/gutenberg/?path=/docs/icons-icon--default

## Testing Instructions




1. `npm run storybook:dev`
2. check http://localhost:50240/?path=/docs/icons-icon--default

## Screenshots or screencast 
![スクリーンショット 2023-06-27 22 03 52](https://github.com/WordPress/gutenberg/assets/487207/4166b047-a44d-4820-90c0-35e018d4ca8d)
